test/integration/dynamic_shared_vpc/controls/svpc.rb

```diff
@@ -157,6 +157,42 @@
         )
       end
     end
+
+    it "service project c with explicit subnets and grant_network_role flag set to false does not include project default service account in the roles/compute.networkUser IAM binding" do
```

Comment:
Lets add also confirm via a test that binding is added for a different service projevt where the flag is true(i.e by default). Maybe https://github.com/terraform-google-modules/terraform-google-project-factory/blob/616ede9456cc8f86ef7995192af3473d17ee7946/examples/shared_vpc/main.tf#L112

Reply:
this test scenario is cover for the [service-project](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/63c7b40d5244d6472d6a22f8767c736987ef7e48/test/integration/dynamic_shared_vpc/controls/svpc.rb#L73-L80)